[2025-09-16T21:17:49.090+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: generate_events_minutely.produce_entities_and_events manual__2025-09-16T21:16:32.369060+00:00 [queued]>
[2025-09-16T21:17:49.103+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: generate_events_minutely.produce_entities_and_events manual__2025-09-16T21:16:32.369060+00:00 [queued]>
[2025-09-16T21:17:49.104+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2025-09-16T21:17:49.131+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): produce_entities_and_events> on 2025-09-16 21:16:32.369060+00:00
[2025-09-16T21:17:49.140+0000] {standard_task_runner.py:57} INFO - Started process 6834 to run task
[2025-09-16T21:17:49.144+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'generate_events_minutely', 'produce_entities_and_events', 'manual__2025-09-16T21:16:32.369060+00:00', '--job-id', '884', '--raw', '--subdir', 'DAGS_FOLDER/generator_dag.py', '--cfg-path', '/tmp/tmpdna_y_jh']
[2025-09-16T21:17:49.147+0000] {standard_task_runner.py:85} INFO - Job 884: Subtask produce_entities_and_events
[2025-09-16T21:17:49.173+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-16T21:17:49.230+0000] {task_command.py:410} INFO - Running <TaskInstance: generate_events_minutely.produce_entities_and_events manual__2025-09-16T21:16:32.369060+00:00 [running]> on host d6e2a7c8b3ee
[2025-09-16T21:17:49.497+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='generate_events_minutely' AIRFLOW_CTX_TASK_ID='produce_entities_and_events' AIRFLOW_CTX_EXECUTION_DATE='2025-09-16T21:16:32.369060+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-16T21:16:32.369060+00:00'
[2025-09-16T21:17:49.508+0000] {conn.py:397} INFO - <BrokerConnection client_id=kafka-python-producer-1, node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.23.0.3', 29092)]>: connecting to kafka:29092 [('172.23.0.3', 29092) IPv4]
[2025-09-16T21:17:49.530+0000] {conn.py:618} INFO - <BrokerConnection client_id=kafka-python-producer-1, node_id=bootstrap-0 host=kafka:29092 <checking_api_versions_recv> [IPv4 ('172.23.0.3', 29092)]>: Broker version identified as 2.6
[2025-09-16T21:17:49.531+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-producer-1, node_id=bootstrap-0 host=kafka:29092 <connected> [IPv4 ('172.23.0.3', 29092)]>: Connection complete.
[2025-09-16T21:17:51.083+0000] {conn.py:397} INFO - <BrokerConnection client_id=kafka-python-producer-1, node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.23.0.3', 29092)]>: connecting to kafka:29092 [('172.23.0.3', 29092) IPv4]
[2025-09-16T21:17:51.084+0000] {conn.py:457} INFO - <BrokerConnection client_id=kafka-python-producer-1, node_id=1 host=kafka:29092 <connected> [IPv4 ('172.23.0.3', 29092)]>: Connection complete.
[2025-09-16T21:17:51.085+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-producer-1, node_id=bootstrap-0 host=kafka:29092 <connected> [IPv4 ('172.23.0.3', 29092)]>: Closing connection. 
[2025-09-16T21:17:54.066+0000] {kafka.py:629} INFO - <KafkaProducer client_id=kafka-python-producer-1 transactional_id=None>: Closing the Kafka producer with 9223372036.0 secs timeout.
[2025-09-16T21:17:54.068+0000] {conn.py:945} INFO - <BrokerConnection client_id=kafka-python-producer-1, node_id=1 host=kafka:29092 <connected> [IPv4 ('172.23.0.3', 29092)]>: Closing connection. 
[2025-09-16T21:17:54.070+0000] {generator_dag.py:34} INFO - Produced 5 messages to Kafka
[2025-09-16T21:17:54.071+0000] {python.py:183} INFO - Done. Returned value was: 5
[2025-09-16T21:17:54.372+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=generate_events_minutely, task_id=produce_entities_and_events, execution_date=20250916T211632, start_date=20250916T211749, end_date=20250916T211754
[2025-09-16T21:17:54.379+0000] {logging_mixin.py:150} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:205 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2025-09-16T21:17:54.901+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2025-09-16T21:17:54.980+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
